cmd/main.go

```diff
@@ -68,6 +73,19 @@ const (
 	keySize               = 16
 )
 
+// LogHook to send a graphite metric for each log entry
+func (h LogHook) Run(e *zerolog.Event, level zerolog.Level, msg string) {
+	metrics, err := h.config.metrics()
```

Comment:
Do I understand correctly it will call metrics() every time we send logs?
If so, it is not good because on every log line we will create our own metrics copy and rewrite one we already assigned, it's resource consuming and probably not correct